### ISSUE TITLE
pretrain model for student?

### ISSUE THREAD

**Issue by yangydeng** [szq0214/MEAL-V2#3]
Hi, thanks for your job.
I was looked into your code and found that the func "_create_model" always load pretrained weight, even it's a student model. 
And I'm curious that dose the student always start with a pretrained?
Thank you.

**Comment by szq0214** [comment]
Hi @yangydeng, yes, as we stated in the paper, we also rely on a good initialization of the student, but if you adjust the learning rate scheduler (using larger initial lr) and train with sufficient epochs, I think you can still obtain decent accuracy without such initialization.
